.github/workflows/build_and_test.yml

```diff
@@ -63,7 +63,7 @@ jobs:
           sudo apt-get install freetds-dev
           python -m pip install --upgrade pip
           pip install -r requirements.txt
-          pip install -r mage_integrations/requirements.txt
```

Comment:
why do we need to make this change?

Reply:
@wangxiaoyou1993 running a integration pipeline doesn't work without this change. I get a module error: https://github.com/mage-ai/mage-ai/actions/runs/6870701238/job/18686125555.

```
Traceback (most recent call last):
  File "/home/runner/work/mage-ai/mage-ai/mage_ai/data_integrations/utils/scheduler.py", line 75, in initialize_state_and_runs
    update_stream_states(pipeline_run, logger, variables)
  File "/home/runner/work/mage-ai/mage-ai/mage_ai/data_integrations/utils/scheduler.py", line 192, in update_stream_states
    from mage_integrations.sources.utils import (
ModuleNotFoundError: No module named 'mage_integrations.sources'
```

Reply:
got it